third_party/nvfuser/csrc/executor.h

```diff
@@ -329,6 +347,9 @@ class TORCH_CUDA_CU_API FusionExecutor : public NonCopyable {
 
   // Profiling support: nvrtc log for debugging
   std::string last_compiler_log_;
+
+  // Profiling support: nvrtc compiled binary
```

Comment:
Is this for supporting profiling?

Reply:
Hmm, I don't know what is the definition of "profiling" in this context. My understanding is, anything that is useful for our developer to learn about perf is "profiling", and the compiled binary is one such thing.

Reply:
I'd disagree. The compiled binary itself doesn't give any "profile". Why not just remove it?

Reply:
Yeah, no problem. I don't have strong opinion on how to call it.